include/swift/Basic/Sanitizers.def

```diff
@@ -0,0 +1,28 @@
+//===--- Sanitizers.def - Swift Sanitizers ß----------------------*- C++ -*-===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2019 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+//
+// This file defines the sanitizers supported by Swift.
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef SANITIZER
+#error "Define SANITIZER prior to including this file!"
+#endif
+
+// SANITIZER(enum_bit, kind, name, file)
+
+SANITIZER(1, Address,   address,    asan)
```

Comment:
Huh. We skipped 0.

(We skipped 0 before your patch too.)

Reply:
Yes, should we change it to start with 0?

Reply:
\*shrug\* Sure. Won't make a difference now, but it'll work better when we get to our 32nd supported sanitizer!

Reply:
😅